src/domain/kaivuilmoitus/Areas.tsx

```diff
@@ -74,10 +83,11 @@ export default function Areas({ hankeData }: Readonly<Props>) {
     fields: applicationAreas,
     append,
     remove,
-  } = useFieldArray<KaivuilmoitusFormValues, 'applicationData.areas'>({
+  } = useFieldArrayWithStateUpdate<KaivuilmoitusFormValues, 'applicationData.areas'>({
```

Comment:
Not sure if this is needed, but since `useFieldArrayWithStateUpdate` was used in hanke areas page, I took it here as well.

Reply:
Probably not needed here.

Reply:
Reverted back to `useFieldArray`